src/main/java/picard/sam/markduplicates/UmiUtil.java

```diff
@@ -36,21 +37,29 @@
 
 class UmiUtil {
 
-    static final String DUPLEX_UMI_DELIMITER = "-";
+    public static final String DUPLEX_UMI_DELIMITER = "-";
+    public static final String TOP_STRAND_DUPLEX = "/A";
+    public static final String BOTTOM_STRAND_DUPLEX = "/B";
+    public static final String CONTIG_SEPARATOR = ":";
+    public static final String UMI_NAME_SEPARATOR = "/";
+    static final String INFERRED_UMI_TAG = "inferredUmi";
```

Comment:
Include the word "TRANSIENT" in the variable name, so folks aren't confused that it's of length>2